reana_workflow_controller/workflow_run_managers/__init__.py

```diff
@@ -21,14 +21,27 @@ class WorkflowRunManager():
         {
             'name': 'SHARED_VOLUME_PATH',
             'value': '/reana'
+        },
+        {
+            'name': 'WDB_SOCKET_SERVER',
```

Comment:
Note that the WDB part needs to be removed here, as it will be properly if-else-d in another PR...